### PR TITLE
avoid the jq/yq dependency

### PR DESCRIPTION
**Pull request by goneri**
Replace `yq` calls by `include_vars` from core.

### CODE REVIEW

File: playbooks/ansible-test-base/run.yaml [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
-    - name: Fetch and install the artifacts
-      import_role:
-        name: deploy-artifacts
-      when: ansible_test_collections | default(False)
+    - when: ansible_test_collections | default(False)
+      block:
+        - name: Fetch and install the artifacts
+          import_role:
+            name: deploy-artifacts
+        - name: Copy the galaxy.yml on the controller
```
Comment (pabelanger) [inline playbooks/ansible-test-base/run.yaml:9]: Can't this be simplified using slurp?

Comment (goneri) [reply playbooks/ansible-test-base/run.yaml:9]: Not really, `slurp` stores in memory, `include_vars` reads from a file.

**Comment by pabelanger** [comment]
we pip install yq in ansible-test-base I think, lets also drop it

**Comment by goneri** [comment]
> we pip install yq in ansible-test-base I think, lets also drop it

Hum, where?

**Comment by pabelanger** [comment]
> > we pip install yq in ansible-test-base I think, lets also drop it
> 
> Hum, where?

Ah, my mistake this was deleted in the PR. Okay, merge when ready

**Comment by goneri** [comment]
Merging, I will keep an eye on the results today.

**Pull request merged**